### PR TITLE
Update release docs

### PR DESCRIPTION
**Pull request by nikku**
Incorporates https://github.com/bpmn-io/internal-docs/issues/180.

### CODE REVIEW

File: docs/.project/ALPHA_RELEASE.md [hunk 1]
```diff
@@ -42,18 +43,16 @@ _To be done after code freeze and prior to the release day to prepare and build
...
+   * [ ] send the reviewed draft to Christopher Rogers to coordinate the publishing
 
 _To be done on release day to announce the release and making it publically available._
 
 * [ ] publish release on [Github Releases](https://github.com/camunda/camunda-modeler/releases)
 * [ ] trigger mirroring of release to https://camunda.org/release/camunda-modeler/ via [Jenkins](https://ci.cambpm.camunda.cloud/job/sideprojects/job/camunda-modeler-desktop-RELEASE/build?delay=0sec)
-* [ ] spread the word
-    * [ ] send [release notice email](https://github.com/bpmn-io/internal-docs/blob/master/camunda-modeler/README.md#release-notice-email)
-    * [ ] tweet
+* [ ] send the [release notice email](https://github.com/bpmn-io/internal-docs/blob/master/camunda-modeler/README.md#release-notice-email) to `alle`
```
Comment (barmac) [inline docs/.project/ALPHA_RELEASE.md:55]: If I understand https://github.com/bpmn-io/internal-docs/issues/180 correctly, we should *not* send the email to `alle`.

**Review by barmac — CHANGES_REQUESTED** [review]
See comment for alpha release.

File: docs/.project/ALPHA_RELEASE.md (hunk 1 above)
Comment (nikku) [reply docs/.project/ALPHA_RELEASE.md:55]: Check the _Applies To_ section, it excludes alpha releases.

Comment (barmac) [reply docs/.project/ALPHA_RELEASE.md:55]: All right!

**Review by barmac — COMMENTED** [review]
Looks good to me. @MaxTru what do you think?

File: docs/.project/ALPHA_RELEASE.md [hunk 2]
```diff
@@ -42,18 +43,16 @@ _To be done after code freeze and prior to the release day to prepare and build
     * [ ] Works on Mac
     * [ ] Works on Windows
 * [ ] trigger QA for fuzzy testing
-* [ ] write blog post for [blog.camunda.org](https://camunda.com/blog/) according to [this process](https://confluence.camunda.com/pages/viewpage.action?spaceKey=MAR&title=Your+Content+on+the+Camunda+Blog)
-   * [ ] copy and paste [this template](https://docs.google.com/document/d/18aXWFBdbET97EZ_HO1DEVLVsYP-UQEabumSzQdeRbDw/edit?usp=sharing) to a new docs file in [Modeling blog posts](https://drive.google.com/drive/u/1/folders/1Kw3lo2zbl3HL7w6y19A3S9SdpAdrDcFJ)
-   * [ ] write a draft version of the blog post
-   * [ ] send it to Charley Mann and Christopher Rogers for review and ask them to coordinate the publishing
+* [ ] write blog post for the [Camunda Blog](https://camunda.com/blog/)
+   * [ ] create a draft post on Wordpress
+   * [ ] send the draft to Charley Mann for review
```
Comment (MaxTru) [inline docs/.project/ALPHA_RELEASE.md:48]: Proposal: write all names consistently using \`<name>\`. E.g., here: `Charley Mann`.

File: docs/.project/RELEASE.md [hunk 3]
```diff
@@ -56,9 +57,7 @@ _To be done on release day to announce the release and making it publically avai
 * [ ] publish release on [Github Releases](https://github.com/camunda/camunda-modeler/releases)
 * [ ] trigger mirroring of release to https://camunda.org/release/camunda-modeler/ via [Jenkins](https://ci.cambpm.camunda.cloud/job/sideprojects/job/camunda-modeler-desktop-RELEASE/build?delay=0sec)
 * [ ] update download version of [Camunda Modeler](https://camunda.com/de/download/modeler/) by reaching out to David Paradis
```
Comment (MaxTru) [inline docs/.project/RELEASE.md:59]: Proposal: write all names consistently using \`<name>\`.

**Review by MaxTru — COMMENTED** [review]
LGTM, only a minor thing for consistency

**Comment by nikku** [comment]
Quoted a few more names. Should be good now.

**Pull request merged**